redbot/cogs/warnings/warnings.py

```diff
@@ -433,14 +439,15 @@ def __init__(self, bot: Red):
                 await member_settings.total_points.set(current_point_count)
                 user_warnings.pop(warn_id)
         try:
+            reason_msg = "{description}".format(description=reason)
```

Comment:
This is unneeded, you can just use the `reason` variable instead of `reason_msg` in `modlog.create_case` call.